modules/understanding-agent-install.adoc

```diff
@@ -9,6 +9,11 @@ As an {product-title} user, you can leverage the advantages of the Assisted Inst
 
 The Agent-based installation comprises a bootable ISO that contains the Assisted discovery agent and the Assisted Service. Both are required to perform the cluster installation, but the latter runs on only one of the hosts.
 
+[NOTE]
+====
+Currently, ISO boot is not supported on {ibm-z-name} (`s390x` architecture). The recommended method is by using PXE assets, which requires specifying additional kernel arguments.
```

Comment:
See note above about whether architecture should, or should not, be in parentheses.

Reply:
Shouldn't